### PR TITLE
chore(deployment): move deployment (helm, d-c) to separate folder

### PR DESCRIPTION
**Pull request by luke-biel**
* move helm to deployment/ dir
* split d-c into several smaller composes
* ensure tests use d-c from deployment

### CODE REVIEW

**Comment by luke-biel** [comment]
I still need to go over documentation (check if it still points into proper places), but rest is ready for review

File: deployment/compose/README.md [hunk 1]
```diff
@@ -16,27 +18,31 @@ For k8s deployment, please refer to our [documentation](../../docs/k8s_local_dep
 
 ## Volume
 
-The directory `./docker-volume` is used as a volume. Please note it is not fully `.gitignore`d because we rely on `init.sql` for postgres, both in docker-compose and in Github Actions. DO NOT REMOVE IT UNDER ANY CIRCUMSTANCES.
+The directory `./docker-volume` is used as a volume. Please note it is not fully `.gitignore`d because we rely on some setup scripts attached via volumes.
```
Comment (luke-biel) [inline deployment/compose/README.md:21]: It was too specific, I don't think we need to go in detal here.

File: deployment/compose/README.md [hunk 2]
```diff
@@ -16,27 +18,31 @@ For k8s deployment, please refer to our [documentation](../../docs/k8s_local_dep
...
-`docker-compose up -d kafka zoo postgres`
+`docker-compose up -d`
 
-After it had some time to setup, you can proceed with rest of the environment:
+If you want to add cdl components to it, you must specify `-f` options:
 
-`docker-compose up -d`
+`docker-compose -f docker-compose.cdl.yml -f docker-compose.yml up -d`
+
+Sometimes it's useful to store data on disk (eg. for debugging), we can achieve this by adding `-f docker-compose.persistent-storage.yml` to combination:
```
Comment (Kononnable) [inline deployment/compose/README.md:37]: It's persistent either way. The difference is if files are saved on docker layer or in parent system filesystem.

**Review by Kononnable — APPROVED** [review]

**Comment by Kononnable** [comment]
We could also change `helm-infrastructure` to `infrastructure` since it's in a `helm` directory.

File: deployment/compose/README.md (hunk 2 above)
Comment (luke-biel) [reply deployment/compose/README.md:37]: Well, it's persistent between runs, was what I meant. Can't think of anything better.

**Comment by luke-biel** [comment]
> We could also change `helm-infrastructure` to `infrastructure` since it's in a `helm` directory.

It is changed

File: deployment/compose/README.md (hunk 2 above)
Comment (Kononnable) [reply deployment/compose/README.md:37]: Same goes for solution without volumes. `docker-compose down` isn't the only solution, there is `docker-compose stop`

Comment (luke-biel) [reply deployment/compose/README.md]: True that. Hmm, I think I have another name.

File: deployment/helm/infrastructure-druid/Chart.yaml [hunk 3]
```diff
@@ -25,4 +25,4 @@ appVersion: 0.0.5
 dependencies:
 - name: druid
   version: "0.2.10"
-  repository: "https://kubernetes-charts-incubator.storage.googleapis.com"
+  repository: "https://charts.helm.sh/incubator"
```
Comment (luke-biel) [inline deployment/helm/infrastructure-druid/Chart.yaml:28]: Hmm, I thought that it's an error within merge that Incubator address wasn't changed. Oh well, this should be separate commit.

**Review by Kononnable — DISMISSED** [review]

**Review by Kononnable — APPROVED** [review]
